os/keycodes.h

```diff
@@ -1,9 +1,15 @@
 #pragma once
 
+// C++ includes
+#include <limits>
+
 namespace GeNNRobotics {
 namespace OS {
-namespace KeyCodes {
+
 #ifdef _WIN32
+static const int KeyMask = std::numeric_limits<int>::max();
```

Comment:
Shouldn't key codes be unsigned?

Reply:
Hmm I dunno. Intuitively yes, but the value you get from `waitKeyEx()` is a signed integer (so that it can return -1 in the case of no key pressed). This way does seem to work though.